generators/server/templates/src/routes/clientRouter.js

```diff
@@ -5,11 +5,22 @@ const router = new express.Router();
 
 <% if(options["authentication"] === "passport") { -%>
 const clientRoutes = ["/", "/user-sessions/new", "/users/new"];
+const authedClientRoutes = ["/authed-path"]
 <% } else { -%>
 const clientRoutes = ["/client", "/name"];
 <% } -%>
 router.get(clientRoutes, (req, res) => {
   res.sendFile(getClientIndexPath());
 });
 
+<% if(options["authentication"] === "passport") { -%>
```

Comment:
this should probably be handled via a middleware, so I'm not sure we want to establish this pattern.